.env

```diff
@@ -48,4 +48,4 @@ HC_RETRIES=3
 HC_START_PERIOD=60s
 
 # Use the local filepath in your computer
-CONFIG_FILEPATH=
+CONFIG_FILEPATH="/Users/egekorkan/git/infrastructure/conf"
```

Comment:
Mhh, I doubt this should be pushed...

@egekorkan

Reply:
Oops